docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,265 @@
+            "Condition": {}
+        }
+    ]
+}
+```
+
+Finally, click "Create role".
+
+## Step 2/4. Configure the Teleport IAM role mapping
+
+The next step is to give your Teleport users permissions to assume IAM roles in
+your Teleport cluster.
+
+You can do this by creating a Teleport role with the `aws_role_arns` field
+listing the IAM role ARN created in the previous step. Create a file called
+`aws-dynamodb-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: aws-dynamodb-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    aws_role_arns:
+    - arn:aws:iam::(=aws.aws_access_key=):role/ExampleTeleportDynamoDBRole
+```
+
+<Details title="Templating aws_role_arns" opened={false}>
+The `aws_role_arns` field supports template variables so they can be populated
+dynamically based on your users' identity provider attributes. See [Role
+Templates](../../access-controls/guides/role-templates.mdx) for details.
+</Details>
+
+Create the new role:
+
+```code
+$ tctl create -f aws-dynamodb-access.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="aws-dynamodb-access"!)
+
+## Step 3/4. Install the Teleport Database Service
+
+Create and EC2 instance to host the Teleport Database Service, and attach the
+`TeleportDatabaseService` to it. If you're hosting the service another way, you
+must provide AWS authentication to the host manually.
+
+<Admonition type="warning" title="non-standard AWS regions">
+For non-standard AWS regions such as AWS GovCloud (US) regions and AWS China
+regions, please set the corresponding region in the `AWS_REGION` environment
+variable or in the AWS credentials file so that the Application Service can use
+the correct STS endpoint.
+</Admonition>
+
+### Generate a token
+
+A join token is required to authorize a Teleport Database Service instance
+to join the cluster. Generate a short-lived join token and save the output of
+the command:
+
+```code
+$ tctl tokens add \
+    --type=db \
+    --db-name=aws-dynamodb
+```
+
+Use the token provided by the output of this command in the next step.
+
+### Install and start Teleport
+
+Install Teleport on the host where you will run the Teleport Application
+Service. See our [Installation](../../installation.mdx) page for options
+besides Linux servers.
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Edit `/etc/teleport.yaml` to configure the database entry:
+
+```yaml
+version: v3
+teleport:
+  nodename: CHANGEME
+  data_dir: /var/lib/teleport
+  proxy_server: teleport.example.com
+  auth_token: (=presets.tokens.first=)
+  ca_pin:
+  - (=presets.ca_pin=)
+db_service:
+  enabled: "yes"
+  # Matchers for database resources created with "tctl create" command.
+  # For more information: https://goteleport.com/docs/database-access/guides/dynamic-registration/
+  resources:
+  - labels:
+      "*": "*"
+  # Lists statically registered databases proxied by this agent.
+  databases:
+    - name: "aws-dynamodb"
```

Comment:
```suggestion
    - name: "example-dynamodb"
```
nit: Change this to a name that matches example commands suggested below. I prefer "example-" over "aws-" because it is more clearly an example Teleport DB name.